### DISCUSSION TITLE
ReEDS Version 2021.1 is now available!

### DISCUSSION THREAD

**Discussion by wesleyjcole** [NREL/ReEDS-2.0#77] [Announcements]
With the release of the 2021 Standard Scenarios [report](https://www.nrel.gov/docs/fy22osti/80641.pdf), the ReEDS model version 2021.1 is now available.  The 2021.1 version was used to generate all the results for the 2021 report.  Table A-4 in the report's appendix summarizes the changes made to the since the 2020 version.  There are quite a few changes, with the largest changes related to the ability to model zero-carbon power systems.

Feel free to reach out if you have questions about the new version, or if you have recommendations for how we should invest our efforts to enhance the model.

**Comment by wanyingw193** [comment]
Hi, that is so exciting! Where could we access the lastest version? Thanks!

**Reply by vcelloho** [reply]
If you pull from the latest version of the master/main branch you will be using the 2021.1 version. I've provided a link to the commit.
https://github.com/NREL/ReEDS_OpenAccess/commit/37ba7fc488a5bdfaeb98152de8940924d43e52c0

**Reply by wanyingw193** [reply]
Thank you!

**Comment by wanyingw193** [comment]
Hi, just wondering is the tech option BECCS referring to dedicated BECCS or co-fire BECCS? Thanks!

**Reply by vcelloho** [reply]
Dedicated BECCS, from Cambium documentation which aligns with ReEDS.
"Bioenergy with carbon capture and storage technologies are assumed to have a net capture rate of 79.6 kg of CO2 per MMBtu of fuel, following the assumption in ReEDS. That value is for CO2 from direct combustion, and the rest of the emission factors take the same values as the biomass category. "

https://www.nrel.gov/docs/fy22osti/81611.pdf

**Reply by wanyingw193** [reply]
Thanks!

**Comment by wanyingw193** [comment]
Hi, when browsing cases.csv, I can't find the associated codes for some of the input switches. For example, when entering GSw_SitingUPV, which gams file should be check to get the relevant code? I have the same problem for GSw_SitingWindOfs, GSw_SitingWindOns, osprey_years, and GSw_MCS.

In addition, what is the reference source for GSw_OffshoreFrc input for offshore wind installation targets? 

Thank you!

**Reply by wesleyjcole** [reply]
Often the most convenient way to understand a switch is to search for it within a directory.  E.g., if you search for GSw_SitingUPV you'll see that it is passed as an input in a_writedata.gms into writesupplycurves.py, cfgather.py, LDC_prep.py, and hourly_process.py.  Then if you looked in writesupplycurves.py you would see that this switch controls the name of the UPV supply curve input file

`upv_in = pd.read_csv(scdir + 'upv_supply_curve_{}bin-{}.csv'.format(numbins_upv,GSw_SitingUPV))`

I hope that helps!

**Comment by wanyingw193** [comment]
Hi, just want to confirm, the coal price in ReEDS is refer to the minemouth or the delivered prices? Thanks!

**Reply by wesleyjcole** [reply]
Yes.  All fuel prices in the model are delivered prices.

**Reply by wanyingw193** [reply]
Thank you!

**Comment by wanyingw193** [comment]
Hi NREL team, I got a little confused about the system cost in the bokehpivot. For the 'system cost annualized', there is 'discounted by year' and 'undiscounted by year'. What is the main difference and application of these options? Thank you in advance for your attention!

**Reply by vcelloho** [reply]
The discounted by year option applies a specified discount rate to the annual costs in the system cost to produce a total system cost for the specified period in terms of a present value.

undiscounted by year in contrast does not apply any discounting and is a direct summation of the annualized costs. The undiscounted option will always produce a higher total cost than the discounted option.

**Reply by wanyingw193** [reply]
Thank you! Are both the discounted and undiscounted options creating system costs in the 2020 dollar year?
